.github/ISSUE_TEMPLATE.md

```diff
@@ -2,43 +2,4 @@
 Thanks for filing an issue 😄 ! Before you submit, please read the following:
 
 Search open/closed issues before submitting since someone might have asked the same thing before!
-
-Issues on GitHub are only related to problems of Arbiter itself and we cannot answer
-support questions here.
--->
-
-Choose one: is this a bug report or feature request?
```

Comment:
If bug, fill out the  boilerplate. If feature request,, skip and just talk about the request.